src/RA_Defs.h

```diff
@@ -144,14 +165,12 @@ extern GetParseErrorFunc GetJSONParseErrorStr;
 #define SIZEOF_ARRAY( ar )	( sizeof( ar ) / sizeof( ar[ 0 ] ) )
 #define SAFE_DELETE( x )	{ if( x != nullptr ) { delete x; x = nullptr; } }
 
-typedef unsigned char	BYTE;
-typedef unsigned long	DWORD;
-typedef int				BOOL;
-typedef DWORD			ARGB;
+
 
 //namespace RA
 //{
 template<typename T>
+_DEPRECATEDR("Replaced with std::clamp")
```

Comment:
this is only called in one place. suggest making the replacement and removing this code.

Reply:
Sounds good to me.